Makefile

```diff
@@ -80,23 +80,72 @@ megalint:  ## Run the mega-linter.
 load-design-system-templates:  ## Load the design system templates
 	./scripts/load-design-system-templates.sh $(DESIGN_SYSTEM_VERSION)
 
-.PHONY: docker-build
-docker-build: load-design-system-templates  ## Build Docker container
-	docker compose pull
+# Docker and docker compose make commands
+
+.PHONY: compose-build
```

Comment:
one for a follow-up PR: I think we're at a point where we want to have a single `.PHONY` declaration at the top of the file rather thant at the top of each command.

Reply:
Yeah I was thinking the same considering we now how a Makefile that's 150 lines long.. 👀 
However the way we do it now makes it easier to maintain as you can quickly add/ confirm that the recipe is marked as PHONY.
So I guess that's up for debate - let's see how the current way works for us and we might change it in the future.